pkgs/applications/misc/gcalcli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, pkgs, lib, python, pythonPackages }:
```

Comment:
- `stdenv` is not used.
- never pass in `pkgs`.
- pass in either `python` or `pythonPackages`, no need for both. You can access `pythonPackages` via `python.pkgs` and the interpreter with `pythonPackages.python`.